ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
Note that the final const must expose both a Set method and a string key. The Set method is used by client code, but the key is used by the tracer to build special handling of these tags. Maybe we should just add a comment at the top of the file showing the intended usage pattern (yes, different PR).